docs/commands.md

```diff
@@ -2,22 +2,22 @@
 
 ## Create
 
-### TS.CREATE
+### `TS.CREATE`
 
 Create a new time-series.
 
 ```sql
-TS.CREATE key [RETENTION retentionSecs] [LABELS field value..]
+TS.CREATE key [RETENTION retentionSecs] [CHUNK_SIZE chunksize] [LABLES <labelName labelValue> ...]
 ```
 
-* key - Key name for timeseries
+* `key` - Key name for a timeseries
 
 Optional args:
 
- * retentionSecs - Maximum age for samples compared to current time (in seconds)
-    * Default: The global retenion secs configuration of the database (by default, `0`)
-    * When set to 0, the series is not trimmed at all
- * labels - Set of key-value pairs that represent metadata labels of the key
+ * `retentionSecs` - Maximum age for samples compared to current time (in seconds)
```

Comment:
in this change it's no longer clear that there is a global module specific default.  that if even not set is 0